geodistpy/geodesic.py

```
@@ -317,6 +317,222 @@ def geodist_dimwise(X):
+    returns a 3-element tuple ``(distance, fwd_azimuth, back_azimuth)``
+    instead of just the scalar distance.
+
+    Parameters:
+        point1 : (latitude_1, longitude_1)
+            The coordinates of the first point in degrees.
+        point2 : (latitude_2, longitude_2)
+            The coordinates of the second point in degrees.
+
+    Returns:
+        (distance, fwd_azimuth, back_azimuth) : (float, float, float)
+            *distance* in **meters**, azimuths in **degrees** (0–360).
```

Ruff (0.15.1)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)